### DISCUSSION TITLE
Support local three.js or not？

### DISCUSSION THREAD

**Discussion by tp1415926535** [rocksdanister/lively#455] [General]
I want to set local html file which use three.js framework to wallpaper, but it doesn't work normal like in browser. Maybe the reason is referenced other local javascripts in the html? I don't know beacuse there is nothing tips besides blank screen.

**Comment by rocksdanister** [comment]
Local html files are supported.

Some of the examples in the gallery use threejs
https://www.deviantart.com/livelywallpaper/gallery/

This demo is also using threejs
https://www.youtube.com/watch?v=B-9g92ixmm0

Can you share your project file ( rightclick wallpaper in gallery -> Export lively .zip) if the license allows it so that I can take a look.
